src/models/source_code_unit.rs

```diff
@@ -468,10 +468,15 @@ impl SourceCodeUnit {
       }
     }
     return Edit::new(
-      Match::new(new_deleted_range, edit.p_match().matches().clone()),
+      Match::new(
+        self.code()[new_deleted_range.start_byte..new_deleted_range.end_byte].to_string(),
+        new_deleted_range,
+        edit.p_match().matches().clone(),
+      ),
       edit.replacement_string().to_string(),
       edit.matched_rule().to_string(),
     );
+    // new_deleted_range
```

Comment:
Um? What's this comment? Wasn't in the previous PR. Typo?

Reply:
addressed.